flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/RestHandlerConfiguration.java

```diff
@@ -78,10 +83,17 @@ public static RestHandlerConfiguration fromConfiguration(Configuration configura
 		final String rootDir = "flink-web-ui";
 		final File webUiDir = new File(configuration.getString(WebOptions.TMP_DIR), rootDir);
 
+		final long metricFetcherUpdateInterval = configuration.getLong(WebOptions.METRIC_FETCHER_UPDATE_INTERVAL);
```

Comment:
missing sanity checks for being > 0.

Reply:
Added sanity check to `MetricFetcherImpl` constructor